lib/ios-log.js

```diff
@@ -51,55 +58,47 @@ class IOSLog {
 
   async startCapture () {
     if (this.udid) { // if we have a real device
-      return await this.startCaptureRealDevice();
+      return this.startCaptureRealDevice();
     }
     // otherwise, if we have a simulator...
-    let xcodeVersion = await xcode.getVersion();
-    let ver = parseInt(xcodeVersion.split('.')[0], 10);
+    let xCodeVersion = await xcode.getVersion(true);
 
-    if (ver < 5) {
-      logger.debug('Starting iOS6.* simulator log capture');
-      this.proc = new SubProcess('tail', ['-f', '-n', '1', '/var/log/system.log']);
+    logger.debug(`Starting iOS ${await this.sim.getPlatformVersion()} simulator log capture`);
+    if (xCodeVersion.major < 5) {
+      this.proc = new SubProcess('tail', ['-f', '-n', '1', SYSTEM_LOG_PATH]);
       await this.finishStartingLogCapture();
       return;
-    }
-
-    let logsPath;
-
-    if (ver >= 6) {
-      logger.debug('Starting iOS 8.*/9.* simulator log capture');
-      if (_.isUndefined(this.simUdid)) {
-        logger.errorAndThrow('iOS 8*/9.* log capture requires a sim udid');
+    } else if (_.isUndefined(this.simUdid)) {
```

Comment:
i think if we have a `return` it can just be `if` instead of `else if`; the latter makes me feel like there's some important reason to have `else` when there's not.